src/components/dashboard/forms/items/AdUnit/AdUnitBasic.js

```diff
@@ -6,73 +6,95 @@ import Grid from '@material-ui/core/Grid'
 import TextField from '@material-ui/core/TextField'
 import Dropdown from 'components/common/dropdown'
 import { constants, schemas, Joi } from 'adex-models'
+import Checkbox from '@material-ui/core/Checkbox'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import { addUrlUtmTracking } from 'helpers/utmHelpers';
 
 const { adUnitPost } = schemas
 
-const AdTypes = constants.AdUnitsTypes
-	.map(type => {
-		return {
-			value: type,
-			label: type.split('_')[1]
-		}
-	})
+const AdTypes = constants.AdUnitsTypes.map(type => {
+	return {
+		value: type,
+		label: type.split('_')[1]
+	}
+})
 
 class AdUnitBasic extends Component {
-
 	constructor(props) {
 		super(props)
-
 		const { newItem } = props
 		this.validateTitle(newItem.title, false)
 		this.validateDescription(newItem.description, false)
 		this.validateTargetUrl(newItem.targetUrl, false)
 		this.validateAndUpdateType(false, newItem.type)
 	}
 
+	componentDidUpdate(prevProps) {
+		if (
+			prevProps.newItem.title !== this.props.newItem.title ||
+			prevProps.newItem.type !== this.props.newItem.type ||
+			prevProps.newItem.temp.addUtmLink !== this.props.newItem.temp.addUtmLink
```

Comment:
If you uncheck the box I should remove the auto added params.
This could be tricky because if the url contains same query params you need to keep the user values.